### DISCUSSION TITLE
Google CLA sign in error as we are not an initial Pull request submitter

### DISCUSSION THREAD

**Discussion by shoaibmushtaq25** [google/android-fhir#463] [Q&A]
Me and @ekigamba are facing an issue on this pull request https://github.com/google/android-fhir/pull/326 as we are not an initial submitter of Pull request but it was Rodgers Andati and he is now more working on it.
The issue details are attached in the picture. Looks like authors need to leave a comment `googlebot I consent.` on the PR.
We would appreciate any help on that. cc: @jingtang10 

![Screenshot from 2021-05-05 13-41-00](https://user-images.githubusercontent.com/12672919/117116389-be5f6a80-ada7-11eb-9d33-2375086b36c9.png)

**Comment by jingtang10** [comment]
Replied in the PR: I can see that `12672919+shoaibmushtaq25@users.noreply.github.com` has signed the CLA but `shoaib.mushtaq@venturedive.com` hasn't and needs to consent. Can you double check @shoaibmushtaq25?

**Comment by shoaibmushtaq25** [comment]
I am not sure what's wrong with it although I already have signed the CLA with my email shoaib.mushtaq@venturedive.com. What should I do to resolve this issue? @jingtang10 ?
![Screenshot from 2021-05-07 15-50-20](https://user-images.githubusercontent.com/12672919/117439750-ad039300-af4c-11eb-9a46-086ef81f4078.png)
